### PR TITLE
Don't try to fetch backup version if no compete backups available

### PR DESCRIPTION
**Pull request by meain**
Regression from https://github.com/alcionai/corso/commit/62daf10213ce29782752924bc1c53a44a65dfe01#diff-f088d3a5f56348ddaf6fbf485c2aeaa6d8b40a860de014f7084db6daf0753829R267

---

#### Does this PR need a docs update or release note?

- [ ] :white_check_mark: Yes, it's included
- [ ] :clock1: Yes, but in a later PR
- [x] :no_entry: No

#### Type of change

- [ ] :sunflower: Feature
- [x] :bug: Bugfix
- [ ] :world_map: Documentation
- [ ] :robot: Supportability/Tests
- [ ] :computer: CI/Deployment
- [ ] :broom: Tech Debt/Cleanup

#### Issue(s)

* #<issue>

#### Test Plan

- [x] :muscle: Manual
- [ ] :zap: Unit test
- [ ] :green_heart: E2E

### CODE REVIEW

File: src/internal/operations/backup.go [hunk 1]
```diff
@@ -241,8 +241,9 @@ func (op *BackupOperation) do(
 	backupID model.StableID,
 ) (*details.Builder, error) {
 	var (
-		reasons         = selectorToReasons(op.Selectors, false)
-		fallbackReasons = makeFallbackReasons(op.Selectors)
+		reasons           = selectorToReasons(op.Selectors, false)
+		fallbackReasons   = makeFallbackReasons(op.Selectors)
+		lastBackupVersion = -1
```
Comment (ryanfkeepers) [inline src/internal/operations/backup.go:246]: there's a const for this now: `NoBackup`

File: src/internal/operations/backup.go [hunk 2]
```diff
@@ -264,9 +265,11 @@ func (op *BackupOperation) do(
 		return nil, clues.Wrap(err, "producing manifests and metadata")
 	}
 
-	_, lastBackupVersion, err := lastCompleteBackups(ctx, op.store, mans)
-	if err != nil {
-		return nil, clues.Wrap(err, "retrieving prior backups")
+	if canUseMetaData {
```
Comment (ryanfkeepers) [inline src/internal/operations/backup.go:268]: The name of this check makes it feel unrelated to the internals.  A comment explaining how this relatets, eg: "if canUseMetaData is false then the conditions are ..." would be helpful.  And maybe a rename of the value.

Comment (pandeyabs) [reply src/internal/operations/backup.go:268]: Keeping this as is to expedite corso fixes. @meain could you please follow up on this? :)

**Review by ryanfkeepers — APPROVED** [review]
